script/Arcueid/Day 8/08_00_ARC08_6.txt

```diff
@@ -409,7 +409,7 @@ Arcueid said she had a specific destination in mind, but hasn't let me in on the
 [sha:05cdbe2f523ea3140f6984887ba602f96a824b77]{
 -- Page 318, Offset 20462.
 -- 「今夜はピンポイントで視てもらうわ。まずは騒がしい方ね」
-"Tonight I'll have you look at some locations I've pinpointed.
+"Tonight, I'll have you look at some locations I've pinpointed.
```

Comment:
Unnecessary imo

Reply:
yeah, this sounds like a Top Gear opening